### ISSUE TITLE
UBRS doors not opening/gyth event restarting over and over

### ISSUE THREAD

**Issue by Nygaardens** [SunstriderEmu/BugTracker#2550]
My buddy and i were trying to clear UBRS to get the ace of beasts from The Beast in UBRS (Needed to obtain one of the BIS trinkets - even in TBC), but the every door we met did not open, and the Gyth boss event kept restarting itself after we finished it.

**Comment by kmayberry91** [comment]
I am also having issues with the first door in UBRS side. We killed all 7 flame sigil rooms and the flame dragons but cant get through the door still.
![image](https://user-images.githubusercontent.com/45275285/77839815-9a117300-713d-11ea-8cc8-0575412648b1.png)

**Comment by SabreValkyrn** [comment]
Can confirm, Dragonspire Hall door does not open even when all the orcs (and Dragonkin) are dead.

Looks like many of the runes are not lit up. I didn't pay attention to what exact order I killed the mobs in. I definitely didn't kill all the mobs while inside the spawn room itself.

From [this image](https://www.almarsguides.com/Almar's%20Stuff/WoW/Farming/10k/Dragonspire%20Hall.jpg) I have marked in red which ones are *not* illuminated.

![Dragonspire Hall](https://user-images.githubusercontent.com/61754281/79289162-6d6b9400-7e7d-11ea-8bbb-edb68b8664e4.jpg)


This makes Darkmoon Card: Blue Dragon impossible to get.
